ReleaseNotes.md

```diff
@@ -8,6 +8,36 @@ information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeo
 additional questions or comments.
 
 ## Release History
+### 05/16/2023 -
+##### Version microsoft/oms:3.1.8 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:3.1.8 (linux)
+##### Version microsoft/oms:win-3.1.8 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-3.1.8 (windows)
+##### Current dependencies
+- Linux
+  - CBL-Mariner 2.0
```

Comment:
lets add exact mariner image version - 2.0.20230426 so that we can better track Mariner vulenrabilities
and  having Mariner release link also will be helpful .

https://github.com/microsoft/CBL-Mariner/releases/tag/2.0.20230426-2.0

Reply:
changed